stat/statmat_test.go

```diff
@@ -478,3 +718,65 @@ func BenchmarkCorrToCov(b *testing.B) {
 		corrToCov(cc, sigma)
 	}
 }
+
+func BenchmarkLassoRegression(b *testing.B) {
```

Comment:
If we're adding benchmarks, they will need to have a timer reset after the set-up, but also the set up doesn't need to do all the work that it is; it should work directly into `x`, and the matrix constructions should happen outside the benchmark loop.